superset/assets/javascripts/explore/stores/controls.jsx

```diff
@@ -240,6 +240,13 @@ export const controls = {
     description: 'Sort bars by x labels.',
   },
 
+  combine_metric: {
+    type: 'CheckboxControl',
+    label: 'Combine Metrics',
+    default: false,
+    description: 'Combine Metrics to display.',
```

Comment:
It's useful to compare data for different scenarios, when there exist multi metrics and they are relevant.

original behavior:
<img width="859" alt="2017-07-09 17 17 57" src="https://user-images.githubusercontent.com/7597155/27992622-8fb8b52a-64cb-11e7-8b79-9db612debd55.png">

after select 'Combine Metrics'
<img width="869" alt="2017-07-09 17 18 35" src="https://user-images.githubusercontent.com/7597155/27992626-a0ecc3b8-64cb-11e7-9cf0-cd2c8cced60a.png">